docs/plugins/purchase.md

```diff
@@ -88,72 +84,32 @@ Purchase.setup('http://csp-backend.com/api', {
     uri: '/subscriptions/:id',
     method: 'DELETE',
   },
-  signature: {
-    uri: '/assets/:id/signature',
-    method: 'POST',
-    headers: {
-      'x-auth-token': 'abcd123xyz'
-      'cache-control': 'no-cache'
-    }
-  },
```

Comment:
I think this example was added afterwards. We should re-add it.

Reply:
Example added.